apps/builder/src/app/mysql/account.ts

```diff
@@ -39,13 +56,27 @@ export const readUser = (): Promise<Transaction[]> => {
         return <Transaction> {
           amount:row.amount,
           date:new Date(row.date),
-          message:row.message, 
-          from:accountMap.get(row.from_account_id),
-          to:accountMap.get(row.to_account_id),
+          message:row.message,
+          to:accountMap.get(row.to_account_id) || null,
         }
-      }))  
+      }))
      });
-      
-    connect.end(); 
   })
 }
+export const readBalance = (account): Promise<Balance[]> => {
+  const queryStringBalance = 'select balance from Account where id = ?';
+  return new Promise<Balance[]>((resolve,reject)=> {
+    connect.query(queryStringBalance,[account], (err, rows) => {
+      if (err) {
+        reject(err)
+        return
+      } ; 
+        resolve(rows.map(row=>{
+          return <Balance> {
+            amount: row.balance,
+            uid:accountMap.get(account) || null,
+          }
+         }))
+     });
+  })
+}
```

Comment:
Kan de komme over i sin egen fil? Du har `transactions.ts` til at håndtere transaktionerne. Du har `balance.ts` til at håndtere balance. Men SQL for begge dele er ind i samme fil.